cmd/zc_filter.go

```diff
@@ -230,6 +231,57 @@ func (fs filterSet) GetEnumerationPreFilter(recursive bool) string {
 
 ////////
 
+// includeRegex & excludeRegex
+type regexFilter struct {
+	patterns      []string
+	matchResponse bool
+}
+
+func (f *regexFilter) doesSupportThisOS() (msg string, supported bool) {
+	msg = ""
+	supported = true
+	return
+}
+
+func (f *regexFilter) appliesOnlyToFiles() bool {
+	return true
+}
+
+func (f *regexFilter) doesPass(storedObject storedObject) bool {
+	if len(f.patterns) == 0 {
+		return true
+	}
+	for _, pattern := range f.patterns {
+		matched := false
+		var err error
+
+		matched, err = regexp.MatchString(pattern, storedObject.name)
+
+		if err != nil {
+			return false
```

Comment:
If the regex parsing failed, perhaps we should let it be matchResponse instead of always false?